tools/add_triage_signature.py

```diff
@@ -1378,6 +1378,39 @@ def _process_ticket(self, url, issue_key):
             self._update_triaging_ticket(report)
 
 
+class MissingMustGatherLogs(ErrorSignature):
+    """
+    This signature add is added in case must-gather logs weren't collected after the failure of the installation.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            **kwargs,
+            function_impact_label="missing_must-gather_logs",
```

Comment:
to follow existing conventions on this label:
```suggestion
            function_impact_label="missing_must_gather_logs",
```